v2/rscg_examples_site/docs/indexRSCG.md

```
@@ -20,6 +20,7 @@ This is the list of 283 ( 16 from Microsoft) RSCG with examples
 
 | No        | Name  | Date | Category |
 | --------- | ----- | ---- | -------- |  
+|284| [Pinecone.TypedPath by Nick Pinecone ](/docs/Pinecone.TypedPath)|2026-08-24 => 24 August 2026 | [FilesToCode](/docs/Categories/FilesToCode) |
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside link text

(MD039, no-space-in-links)